server.js

```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+
+var mongoose = require('mongoose');
+
+var rcCollectionRouter = require('./rcCollectionroutes');
+
+var rcRecordRouter = require('./rcRecordroutes');
+
+var app = express();
+
+var db = mongoose.connect('mongodb://localhost/reCollector');
+
+app.use(express.static('public'));
+
+app.use('/rcCollection', rcCollectionRouter);
+
+app.use('/rcRecord', rcRecordRouter)
+
+app.use('/serverModule', router);
```

Comment:
I don’t think it’s necessary, artifact from development . I removed it and pushed the change.

From: jamdon 
Sent: Friday, June 17, 2016 1:52 AM
To: JSCRIPT300-Spring2016/Final_Project 
Cc: seancava ; Author 
Subject: Re: [JSCRIPT300-Spring2016/Final_Project] Final Project Proposal (#10)

In server.js:

> +
> +var rcCollectionRouter = require('./rcCollectionroutes');
> +
> +var rcRecordRouter = require('./rcRecordroutes');
> +
> +var app = express();
> +
> +var db = mongoose.connect('mongodb://localhost/reCollector');
> +
> +app.use(express.static('public'));
> +
> +app.use('/rcCollection', rcCollectionRouter);
> +
> +app.use('/rcRecord', rcRecordRouter)
> +
> +app.use('/serverModule', router);
router is undefined - does this need to be required in, or is this line even necessary?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.